src/search.ts

```diff
@@ -170,13 +170,13 @@ export class Search {
 
                 let label = new St.Label({
                     text: title,
-                    styleClass: "pop-shell-search-label",
+                    styleClass: "pop-shell-search-label list-search-result-title",
                     y_align: Clutter.ActorAlign.CENTER
                 });
 
                 label.clutter_text.set_ellipsize(Pango.EllipsizeMode.END);
 
-                let container = new widgets.Box({ styleClass: "pop-shell-search-element" })
+                let container = new widgets.Box({ styleClass: "pop-shell-search-element list-search-result" })
```

Comment:
This looks better in Pop and Adwaita, but in many themes it causes the text to become unreadable. I think the existing styling present for these widgets was fairly theme-agnostic and resulted in guaranteed-readable text.